src/test/com/exeptions/MyExeptions.java

```diff
@@ -0,0 +1,46 @@
+package test.com.exeptions;
```

Comment:
"exCeptions", everything should be neat and tidy and grammatically correct. Otherwise it's a sloppy work